Integrations/integration-Awake_Security.yml

```diff
@@ -0,0 +1,607 @@
+
+        if "lookback_minutes" in args:
+            request["lookback_minutes"] = int(args["lookback_minutes"])
+        else:
+            request["lookback_minutes"] = 480
+
+        response = requests.post(prefix + path, json=request, headers=headers, verify=verify)
+
+        return response.json()
+
+    def lookupDevice(lookup_key):
+        contents = lookup("device", lookup_key)
+
+        humanReadableFields = [
+            "deviceScore",
+            "deviceName",
+            "deviceType",
+            "os",
+            "osVersion",
+            "commonEmail",
+            "commonUsername",
+            "tags",
+            "recentIP",
+            "activeIP",
+            "nSimilarDevices",
+            "ipCount",
+            "applicationCount",
+            "protocols",
+            "firstSeen",
+            "lastSeen",
+        ]
+
+        if "deviceScore" in contents:
+            dbotScore = toDBotScore("device", contents["deviceScore"], lookup_key)
+        else:
+            dbotScore = None
+
+        humanReadable = displayTable([contents], humanReadableFields)
+
+        returnResults(contents, humanReadable, dbotScore)
+
+    def lookupDomain(lookup_key):
+        contents = lookup("domain", lookup_key)
+
+        humanReadableFields = [
+            "notability",
+            "isAlexaTopOneMillion",
+            "isDGA",
+            "intelSources",
+            "numAssociatedDevices",
+            "numAssociatedActivities",
+            "approxBytesTransferred",
+            "protocols",
+            "firstSeen",
+            "lastSeen",
+        ]
+
+        if "notability" in contents:
+            dbotScore = toDBotScore("domain", contents["notability"], lookup_key)
+        else:
+            dbotScore = None
+
+        humanReadable = displayTable([contents], humanReadableFields)
+
+        returnResults(contents, humanReadable, dbotScore)
+
+    def lookupEmail(lookup_key):
+        contents = lookup("email", lookup_key)
+
+        humanReadableFields = [
+            "notabilityPercentile",
+            "deviceName",
+            "os",
+            "deviceType",
+            "application",
+            "numberSimilarDevices",
+            "numberSessions",
+            "firstSeen",
+            "lastSeen",
+            "duration",
+            "deviceId",
+        ]
+
+        if "notabilityPercentile" in contents:
+            dbotScore = toDBotScore("email", contents["notabilityPercentile"], lookup_key)
+        else:
+            dbotScore = None
+
+        humanReadable = displayTable(contents, humanReadableFields)
+
+        returnResults(contents, humanReadable, dbotScore)
+
+    def lookupIp(lookup_key):
+        contents = lookup("ip", lookup_key)
+
+        humanReadableFields = [
+            "deviceCount",
+            "activityCount",
+            "ipFirstSeen",
+            "ipLastSeen",
+        ]
+
+        # Note: No DBotScore for IP addresses as we do not score them.
```

Comment:
we still want to indicate that we enriched the ip, even if we can't generate a score using awake, so here dbotscore 0 is a good option